09_dt_rtc/patch/0001-task09-Done.patch

```diff
@@ -0,0 +1,104 @@
+From 822cec0e07eaece5e126b71611afab91dc87252e Mon Sep 17 00:00:00 2001
+From: alexposukhov <33048624+alexposukhov@users.noreply.github.com>
+Date: Mon, 15 Mar 2021 09:16:23 +0200
+Subject: [PATCH] task09: Done
+
+Signed-off-by: Anton Kotsiubailo <antohakotsubailo@gmail.com>
+---
+ 09_dt_rtc/README.md | Bin 0 -> 4065 bytes
+ 09_dt_rtc/log.txt   | Bin 0 -> 7948 bytes
+ scissors/README.md  | Bin 481 -> 0 bytes
+ 3 files changed, 0 insertions(+), 0 deletions(-)
+ create mode 100644 09_dt_rtc/README.md
+ create mode 100644 09_dt_rtc/log.txt
+ delete mode 100644 scissors/README.md
```

Comment:
Please provide correct patch for devicetree.
It should be human readable.
No binary data here.